### ISSUE TITLE
Problème Authentification après Reboot

### ISSUE THREAD

**Issue by kenbest69** [cxii-dev/script.tvshowtime#46]
Bonjour, 
Tout d'abord merci pour cette addon. Mais depuis peu j'ai un soucie, lorsque j'installe l'addon et me connecte tout marche parfaitement mais une fois kodi redémarrer impossible de se connecter, il y a "balise user" qui est vide je dois à chaque redémarrage désinstaller et réinstaller l'addon pour que sa remarche. 

Merci pour ton aide.

**Comment by cxii-dev** [comment]
Bonjour,

Bizarre ça car je l'utilise tous les jours et je redémarre 1x par semaine et aucun soucis ...
Je ne m'en occupe même plus, il est derrière sans affichage et je reçois les notifications de badge parfois par surprise sur mon smartphone ... 

Je ne sais pas si quelqu'un nous lisant a déjà eu ce problème ...

**Comment by kenbest69** [comment]
Oui, vraiment bizarre car sa fonctionnait nickel et du jour au lendemain plus rien
